recipes-asteroid/asteroid-launcher/asteroid-launcher_git.bb

```diff
@@ -23,4 +29,14 @@ do_install_append() {
     install -d ${D}/usr/lib/systemd/user/default.target.wants/
     cp ../asteroid-launcher.service ${D}/usr/lib/systemd/user/
     ln -s ../asteroid-launcher.service ${D}/usr/lib/systemd/user/default.target.wants/asteroid-launcher.service
+
+    install -d ${D}/etc/systemd/system/user@.service.d/
+    cp ../localeEnv.conf ${D}/etc/systemd/system/user@.service.d/locale.conf
+
+    install -d ${D}/var/lib/environment/ceres/
+    cp ../locale.conf ${D}/var/lib/environment/ceres/locale.conf
+    chown 1000:1000 ${D}/var/lib/environment/ceres/locale.conf # ceres:ceres
```

Comment:
The issue is that bitbake creates this file as root, which makes the asteroid-settings app unable to write the selected language to file. However, changing the permissions to ceres is probably wrong anyway, as any app could set environment variables then. Is there some way to only give asteroid-settings write access to this file?